start-docker-compose.sh

```diff
@@ -58,7 +58,6 @@ while getopts ':dtzblh' OPTION; do
     b)
       startZac=true
       buildZac=true
-      localZac=true
```

Comment:
Is this by default true? Or why has it been removed? Otherwise, it might be useful purely as clarification that this is using the local version of Zac?

Reply:
Ah I had to read back through the file, don't worry about it 😄

Reply:
It's `false` by default. I also thought it's a good idea to have it set to true but that makes the next checks (for -b and -l set simultaneously) pretty hard to implement